Scripts/Ports/python3/Setup.local.in

```diff
@@ -0,0 +1,49 @@
+############################################################################
+# This is a surgical list of modules that need to be built
+# into the Pythoncore for Plasma to be happy.
+*static*
+
+_contextvars _contextvarsmodule.c
+_csv _csv.c
+_datetime _datetimemodule.c
+_json _json.c
+_pickle _pickle.c
+_queue _queuemodule.c
+_random _randommodule.c
+_statistics _statisticsmodule.c
+_struct _struct.c
+_typing _typingmodule.c
+array arraymodule.c
+binascii binascii.c
+cmath cmathmodule.c
+math mathmodule.c
+select selectmodule.c
+
+# The _blake2 module depends on libb2 on ARM. This is a pain to propagate downstream, so
+# we'll skip building this one into the core. Nothing in the stdlib seems to require it, so
+# hopefully its absence will go unnoticed.
```

Comment:
The `_blake2` module is used by `hashlib` to implement the `blake2b` and `blake2s` hash functions. It looks like `hashlib` gracefully handles missing hash implementation modules, so hopefully this is safe as long as nobody tries to use the BLAKE2 hashes. (But just to be sure, someone should try to `import hashlib` in the console.)

Reply:
Unfortunately, `hashlib` always depends on the libb2 implementation in the `_blake2` module, so `hashlib` won't import at all on ARM macs. `plPythonPack` imports all modules, so, while `hashlib` doesn't work on arm macs, this doesn't appear to break any of the game scripts.